projects/meta_cl/experiments/anml_replicate.py

```diff
@@ -111,7 +111,7 @@ def __call__(self, pil_image):
     num_meta_testing_runs=10,
 
     # Run through meta-test testing 5 images at a time. No training occurs here.
-    test_test_batch_size=5,
+    test_test_batch_size=1,
```

Comment:
Do we still need to leave it as 1, can we ramp it up to 5 again now we have InstanceNorm2d? Theoretically won't impact the results, and can make it faster.